change/@ni-nimble-components-f4992b1c-b7f0-46e1-a684-b5afff8683f6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Putting a comment as a placeholder as still needs review. I think we should hold off merging this until after the release branch for 2025-07 is created.